tests/test_retrievers_langchain_pathway.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -x
+
+WORKPATH=$(dirname "$PWD")
+LOG_PATH="$WORKPATH/tests"
+ip_address=$(hostname -I | awk '{print $1}')
+function build_docker_images() {
+    cd $WORKPATH
+
+    cd comps/vectorstores/langchain/pathway
+
+    docker build --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -t vectorstore-pathway .
+
+    cd $WORKPATH
+    # --no-cache
+    docker build -t opea/retriever-pathway:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/retrievers/langchain/pathway/docker/Dockerfile .
+    if [ $? -ne 0 ]; then
+        echo "opea/retriever-pathway built fail"
+        exit 1
+    else
+        echo "opea/retriever-pathway built successful"
+    fi
+}
+
+function start_service() {
+    cd $WORKPATH
+
+    # tei endpoint
+    tei_endpoint=5008
+    model="BAAI/bge-base-en-v1.5"
+    docker run -d --name="test-comps-retriever-tei-endpoint" -e http_proxy=$http_proxy -e https_proxy=$https_proxy -p $tei_endpoint:80 -v ./data:/data --pull always ghcr.io/huggingface/text-embeddings-inference:cpu-1.2 --model-id $model
+
+    sleep 30s
+    export TEI_EMBEDDING_ENDPOINT="http://${ip_address}:${tei_endpoint}"
+
+    test_embedding=$(python -c "import random; embedding = [random.uniform(-1, 1) for _ in range(768)]; print(embedding)")
+
+    result=$(http_proxy=''
+    curl $TEI_EMBEDDING_ENDPOINT     -X POST     -d '{"inputs":"Hey,"}'     -H 'Content-Type: application/json')
+
+    echo "embed_result:"
+    echo $result
+
+    sleep 30s
+
+    # pathway
+    export PATHWAY_HOST="0.0.0.0"
+    export PATHWAY_PORT=5432
+
+    docker run -d --name="test-comps-vectorstore-pathway" -e PATHWAY_HOST=${PATHWAY_HOST} -e PATHWAY_PORT=${PATHWAY_PORT} -e TEI_EMBEDDING_ENDPOINT=${TEI_EMBEDDING_ENDPOINT} -e http_proxy=$http_proxy -e https_proxy=$https_proxy -v $WORKPATH/comps/vectorstores/langchain/pathway/README.md:/app/data/README.md -p ${PATHWAY_PORT}:${PATHWAY_PORT} --network="host" vectorstore-pathway
+
+    sleep 45s
+
+    export PATHWAY_HOST=$ip_address  # needed in order to reach to vector store
+
+    docker run -d --name="test-retriever-pathway" -p 5009:7000 -e PATHWAY_HOST=${PATHWAY_HOST} -e PATHWAY_PORT=${PATHWAY_PORT} -e http_proxy=$http_proxy -e https_proxy=$https_proxy opea/retriever-pathway:latest
```

Comment:
Please use `test-comps-retriever-pathway`, so the workflow can clean up all the containers even the test failed.